### ISSUE TITLE
Loading zone "rotatable"

### ISSUE THREAD

**Issue by TRIDENTE-83** [loki79uk/FS25_UniversalAutoload#37]
### For a later update

It would be nice, if the loading zone could also be rotated/tilted over time. 
This could then also be used for very special mod trailers. 😅

![al](https://github.com/user-attachments/assets/c6ea3990-e4f4-4992-a2c6-737c93ac42df)

Thank you Loki for this amazing work.
Best regards, Tridente

**Comment by loki79uk** [comment]
Good call, I will add it.  I've never seen anything before that isn't flat!
